compiler/rustc_error_messages/locales/en-US/parser.ftl

```diff
@@ -1,3 +1,6 @@
 parser-struct-literal-body-without-path =
     struct literal body without path
     .suggestion = you might have forgotten to add the struct literal inside the block
+parser-maybe-report-ambiguous-plus =
```

Comment:
massive nitpick: could you seperate the messages by a newline, that's how I've got it formatted in the `typeck.ftl` file :)

Reply:
Are there autoformatters or style checkers for ftl files?

Reply:
Not that I can find from a quick search, @Manishearth - do you know of any?

Reply:
not that I know of